### PR TITLE
Convert to DEB822 sources on upgrades to Groovy

### PR DESCRIPTION
**Pull request by isantop**
Adds a small conversion item in the postinst that will remove Ubuntu/Pop Sources from the sources.list file and add a default system.sources file in sources.list.d

### CODE REVIEW

File: debian/pop-default-settings.postinst [hunk 1]
```diff
@@ -99,8 +107,22 @@ case "$1" in
             echo "${ISO_KEY}" | apt-key add -
             echo "${SOURCE}" > /etc/apt/sources.list.d/pop-os-apps.sources
         fi
-        ;;
 
+        # Convert system sources from one-line format to DEB822, if present
+        # Generous matching, be more specific if found
+        if grep -e "ubuntu.com" -e "apt.pop-os.org/ubuntu" /etc/apt/sources.list
```
Comment (jackpot51) [inline debian/pop-default-settings.postinst]: You can add /ubuntu to all the ubuntu.com lines just like on apt.pop-os.org/ubuntu

**Review by jackpot51 — CHANGES_REQUESTED** [review]

File: debian/pop-default-settings.postinst (hunk 1 above)
Comment (isantop) [reply debian/pop-default-settings.postinst]: Updated.

**Review by jackpot51 — DISMISSED** [review]
I approve. But with things like this, it is good to make very certain there aren't false positives. So @pop-os/quality-assurance should take a look

**Review by leviport — DISMISSED** [review]
Seems to be working as expected after the sed fixes. Here are the contents of the resulting `sources.list` file on a fresh-ish Focal install that was upgraded to Groovy: 
```
# Ubuntu Repositories





# Pop!_OS Repositories

deb http://ppa.launchpad.net/system76/pop/ubuntu groovy main

deb http://apt.pop-os.org/proprietary groovy main
```

I also tested with a copy of my sources.list file and Jacob's sources.list file, and the Ubuntu sources were removed as expected. I can't think of any potential false positives, so I think this is good to go.

**Review by jacobgkau — APPROVED** [review]
One observation, this re-enables proprietary if it's been disabled in legacy format (e.g. via Repoman on 20.04). I'm considering that expected behavior because pop-upgrade already does the same thing, and this is only going to happen once, during the upgrade. (If/when pop-upgrade is updated to handle deb822 instead of wiping everything and populating the default sources.list in legacy format, this may be something to think through further.)

**Review by jackpot51 — APPROVED** [review]

**Pull request merged**